packages/evm/src/opcodes/util.ts

```diff
@@ -173,12 +173,17 @@ export function subMemUsage(runState: RunState, offset: bigint, length: bigint,
 
   const newMemoryWordCount = divCeil(offset + length, BIGINT_32)
   if (newMemoryWordCount <= runState.memoryWordCount) return BIGINT_0
-
   const words = newMemoryWordCount
-  const fee = common.param('gasPrices', 'memory')
-  const quadCoeff = common.param('gasPrices', 'quadCoeffDiv')
-  // words * 3 + words ^2 / 512
-  let cost = words * fee + (words * words) / quadCoeff
+
+  let cost = BIGINT_0
```

Comment:
I don't think this needs to be instantiated with `BIGINT_0`, this should work 
```suggestion
  let cost: BigInt
```

TypeScript can infer after this if/else statement that `cost` is set (and thus not `undefined`) (AFAIK, but I think I had this same pattern today in another PR)

Reply:
Yes, works! Will update! 🙂